launcher/image/test/test_basic.sh

```diff
@@ -0,0 +1,48 @@
+ #!/bin/bash
```

Comment:
we should call this something like `test_workload.sh` or `test_launcher_workload.sh`

Reply:
changed to test_launcher_workload.sh